client/src/components/Welcome.tsx

```diff
@@ -87,8 +87,8 @@ const Welcome = (): JSX.Element => {
             <div className={`rounded-br-2xl ${commanStyles}`}>Blockchain</div>
           </div>
         </div>
-        <div className="flex flex-col flex-1 items-center justify-start w-full md:mt-0 mt-10">
-          <div className="p-3 justify-end items-start flex-col rounded-xl h-48 sm:w-80 w-full my-5 eth-card white-glassmorphism hover:shadow-2xl shadow-cyan-600">
+        <div className="flex flex-col flex-1 items-end justify-start w-full md:mt-0 mt-10  ">
```

Comment:
Well I want to change this `items-end` property to `items-center`

Reply:
about that i thing in large screens its gud or else the the section below looks missAlign with it ! if you wish i could change it back to center

Reply:
@IrfanshaikH-7,you can create an issue and create a **PR** for this issue as with center it will look much better 
![image](https://github.com/Yana-Gupta/trade-off/assets/103108630/c8eb5ab0-8af1-43ca-8fc6-7ecff1cb2c31)

Btw thanks for the work @IrfanshaikH-7 🥇

Reply:
sure I'll do that now!